src/index.html

```diff
@@ -1,14 +1,57 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css">
+    <title>Moyo</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__wrapper">
+        <a href="#home" class="logo__link">
+          <img
+            class="logo__icon"
+            src="./images/logo.png"
+            alt="logo"
+        >
```

Comment:
https://mateacademy.slack.com/archives/C050SHYFFR9/p1681912970291919

Reply:
also, let's be even more specific in your `alt` attribute - what logo **exactly**?